bindings/wasm/bindings.js

```diff
@@ -0,0 +1,143 @@
+var _ManifoldInitialized = false;
+Module.setup = function() {
+  if (_ManifoldInitialized) return;
+  _ManifoldInitialized = true;
+
+  function toVec(vec, list, f = x => x) {
+    for (let x of list) {
+      vec.push_back(f(x));
+    }
+    return vec;
+  }
+
+  function polygons2vec(polygons) {
+    return toVec(
+        new Module.Vector2_vec2(), polygons,
+        poly => toVec(new Module.Vector_vec2(), poly, p => {
+          if (p instanceof Array) return {x: p[0], y: p[1]};
+          return p;
+        }));
+  }
+
+  Module.geometry2mesh = function(geometry) {
+    const mesh = {
+      vertPos: new Module.Vector_vec3(),
+      vertNormal: new Module.Vector_vec3(),
+      triVerts: new Module.Vector_ivec3(),
+      halfedgeTangent: new Module.Vector_vec4()
+    };
+    const temp = new THREE.Vector3();
+    const p = geometry.attributes.position;
+    const n = geometry.attributes.normal;
+    const x = geometry.index;
+    for (let i = 0; i < p.count; i++) {
+      temp.fromBufferAttribute(p, i);
+      mesh.vertPos.push_back(temp);
+      temp.fromBufferAttribute(n, i);
+      mesh.vertNormal.push_back(temp);
+    }
+    for (let i = 0; i < x.count; i += 3) {
+      mesh.triVerts.push_back(x.array.subarray(i, i + 3));
+    }
+    return mesh;
+  };
+
+  function mesh2geometry(mesh) {
```

Comment:
I'm not sure it's a good idea to codify these I/O functions in the bindings yet. Note my other comment about the two versions I already had to make. And it's going to get a lot more complicated when I add in texture support.

Reply:
I'd also prefer to keep anything `THREE`-specific separate from these other generic bindings.